ingest/scripts/call_loculus.py

```diff
@@ -263,18 +300,15 @@ def submit_or_revise(
     url = f"{organism_url(config)}/{endpoint}"
 
     metadata_lines = len(Path(metadata).read_text(encoding="utf-8").splitlines()) - 1
-    logger.info(f"{logging_strings["gerund"]} {metadata_lines} sequence(s) to Loculus")
+    logger.info(f"{logging_strings['gerund']} {metadata_lines} sequence(s) to Loculus")
 
     params = {
         "groupId": group_id,
     }
     if mode == "submit":
         params["dataUseTermsType"] = "OPEN"
 
-    response = post_fasta_batches(
-        url, sequences, metadata, config, params=params, chunk_size=60000
-    )
-    logger.debug(f"{logging_strings["noun"]} response: {response.json()}")
+    response = post_fasta_batches(url, sequences, metadata, config, params=params, chunk_size=1000)
```

Comment:
I decreased this so that the previews test batching but we should increase it again before merging
```suggestion
    response = post_fasta_batches(url, sequences, metadata, config, params=params, chunk_size=10000)
```